### PR TITLE
Auto Key Cipher Algorithm in C

### PR DESCRIPTION
**Pull request by ravireddy07**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

- Auto Key Cipher Algorithm code added in C and updated README.md
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#6655 
(Write your answer here.)

### CODE REVIEW

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top

**Comment by ravireddy07** [comment]
> Few changes:
> 
> * Add Multi-Line Comments
> * Add a beginner-friendly description at the top

Hey @ankitaggarwal23,
The requested changes have been made, have a look at once.

File: C/cryptography/AutoKey_Cipher.c [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    Enter the length of Plain Text: 5
+    Enter the Plain Text: hello
+
+
+Sample Output:
+    The Plain Text is: hello
+    The Text after Encryption(Cipher text) is: slpwz
+
+After Decryption the Text is: hello
+*/
```
Comment (ankitaggarwal23) [inline C/cryptography/AutoKey_Cipher.c]: shift this at the end of the file

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C/cryptography/AutoKey_Cipher.c (hunk 1 above)
Comment (ravireddy07) [reply C/cryptography/AutoKey_Cipher.c]: Moved to End of file.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**